psi4/CMakeLists.txt

```diff
@@ -157,6 +157,32 @@ get_property(_loc TARGET Libxc::xc PROPERTY LOCATION)
 list(APPEND _addons ${_loc})
 message(STATUS "${Cyan}Using Libxc${ColourReset}: ${_loc} (version ${Libxc_VERSION})")
 
+# <<< The whole xtensor stack >>>
+
+find_package(xtl 0.6.4 CONFIG REQUIRED)
+get_property(_loc TARGET xtl PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
+list(APPEND _addons ${_loc})
+message(STATUS "${Cyan}Using xtl${ColourReset}: ${_loc} (version ${xtl_VERSION})")
+
+find_package(xsimd 7.2.3 CONFIG REQUIRED)
+get_property(_loc TARGET xsimd PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
+list(APPEND _addons ${_loc})
+message(STATUS "${Cyan}Using xsimd${ColourReset}: ${_loc} (version ${xsimd_VERSION})")
+
+find_package(xtensor 0.20.7 CONFIG REQUIRED)
+get_property(_loc TARGET xtensor PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
+list(APPEND _addons ${_loc})
+message(STATUS "${Cyan}Using xtensor${ColourReset}: ${_loc} (version ${xtensor_VERSION})")
+
+find_package(xtensor-blas 0.16.1 CONFIG REQUIRED)
+message(STATUS "${Cyan}Using xtensor-blas${ColourReset}: ${_loc} (version ${xtensor-blas_VERSION})")
+
+find_package(xtensor-python 0.23.1 CONFIG REQUIRED)
+get_property(_loc TARGET xtensor-python PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
+list(APPEND _addons ${_loc})
+message(STATUS "${Cyan}Using xtensor-python${ColourReset}: ${_loc} (version ${xtensor-python_VERSION})")
+find_package(NumPy REQUIRED)
```

Comment:
may want to double-check versions in this block, too

Reply:
Done.